medusa/init/filesystem.py

```diff
@@ -11,6 +11,7 @@
 import tempfile # noqa # pylint: disable=unused-import
 from builtins import map
 
+import tempfile # noqa # pylint: disable=unused-import
```

Comment:
@medariox @sharkykh merging master back into develop, adds this again. Should we add one more commit, to remove it again?

Reply:
Yes. This is the best solution IMO